spk/shairport/src/dsm-control.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+
+# Others
+SHAIRPORT="${SYNOPKG_PKGDEST}/bin/shairport"
```

Comment:
Define `INSTALL_DIR` like this: https://github.com/SynoCommunity/spksrc/blob/master/spk/tt-rss/src/installer.sh#L8
Then, use `${INSTALL_DIR}` instead of `${SYNOPKG_PKGDEST}` throughout this file. Check how tt-rss does it (although most packages use this approach, tt-rss is just an example)